src/box2dx/Box2D.NetStandard/Dynamics/Joints/Weld/WeldJointDef.cs

```diff
@@ -3,10 +3,6 @@
 
 namespace Box2D.NetStandard.Dynamics.Joints.Weld {
   public class WeldJointDef : JointDef {
-    public WeldJointDef()
-    {
-      Type = JointType.WeldJoint;
-    }
 
     [Obsolete("Use Joint.AngularStiffness to get stiffness & damping values",true)]
```

Comment:
@HughPH Yo, why "true"? This causes an error in the build process and a build failure.

Reply:
Hmm, it's okay. Actually only EdgeShape::Set() throws an error.